index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins&display=swap"
+      rel="stylesheet"/>
+    <script
+      src="https://kit.fontawesome.com/79b3da0f3c.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="unit">
+        <header>
+            <nav class="navmenu">
+                <a href="">
+                    <p class="Logo">John Doe</p>
+                </a>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </header>
+        <main class="main-frag">
+            <section class="headline-frag">
+                <div class="headline">
+                    <h1 class="headline1">Hey There. I'm Alex</h1>
+                </div>
+                <p class="headline-phrase">I'm a Software Developer</p>
+                <p class="detail">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
```

Comment:
- [OPTIONAL]  I think it would be better to use  hx (`<h2>`, `<h3>`, ....) instead of  `<p>` tag here 
`<p class="headline-phrase">I'm a Software Developer</p>` Because it's a subheading/sub-title, not a paragraph 💯

> This way will be semantically correct and the screen readers and other technologies will thank you.